docs/localnet.md

```diff
@@ -0,0 +1,54 @@
+# Local Testnets
+
+Prerequisites:
+```bash
+make build
+```
+
+## Single node deployment
+
+To run IPC in the local rootnet just perform the following :
+```bash
+cargo make --makefile ./infra/Makefile.toml node
+
+```
+
+It will create three docker containers (cometbft, fendermint, and eth-api).
+
+To stop run the following:
+```bash
+cargo make --makefile ./infra/Makefile.toml node-down
+```
+
+## Local 4-nodes deployment
+To run IPC in the local rootnet with 4 nodes perform the following command :
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet
+
+```
+
+To stop the network:
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet-down
+```
+
+The testnet contains four logical nodes. Each node consists of cometbft, fendermint, and ethapi containers.
+The testnet network is `192.167.10.0/24`.
+
+ETH-API is accessible on the following interfaces:
+- `192.167.10.10:8545`
+- `192.167.10.11:8546`
+- `192.167.10.12:8547`
+- `192.167.10.13:8548`
```

Comment:
Why does both the IP _and_ the port change? If it's possible to assign different IPs to different nodes, yhy not keep the port be the same on all of them?

Reply:
It is a misprint. In the docker-compose, the ports are the same for containers but different for the host machine. Fixed

Reply:
Well I suppose these addresses are valid on the host, no, and there I assume the ports do have to be different, even though the IPs are also unique?

Reply:
Fixed

Reply:
Oh I see, so the IPs are internal too? Huh, I wonder why they bothered with IPs instead of just using the container names as hostnames (maybe it needs setting the `hostname` of the container, can't remember), which is more readable because you don't have to remember which IP range you are looking at (like 0-3 is fendermint, 10-13 is ethereum, etc, just use `ethapi-0`).

Reply:
You can do both inside the internal network. You can use `cometbft-node0`,...

Reply:
Fixed